pushy/src/main/java/com/eatthepath/pushy/apns/util/ApnsPayloadBuilder.java

```diff
@@ -715,6 +729,76 @@ public ApnsPayloadBuilder addCustomProperty(final String key, final Object value
         return this;
     }
 
+    /**
+     * <p>Sets the provided content state object inside content-state payload for Live Activities.</p>
```

Comment:
Trivial (and reasonable minds might disagree):

```suggestion
     * <p>Sets the content state object inside content-state payload for Live Activities.</p>
```